backport.py

```diff
@@ -22,9 +22,9 @@ def ask_prompt(text):
     print("",file=sys.stderr)
     return reply
 
-merge_re = re.compile('^Merge (#[0-9]+)')
+merge_re = re.compile('^Merge (bitcoin/bitcoin#[0-9]+|bitcoin-core/gui#[0-9]+)')
```

Comment:
It might make sense to use a more general regexp here; e.g. `'^Merge (\w+/\w+#[0-9]+)'`

Reply:
Thanks! Updated.

Reply:
hmm, it doesn't work for "bitcoin-core".

Reply:
Another [update](https://github.com/bitcoin-core/bitcoin-maintainer-tools/pull/105#issuecomment-884181386) should work.

Reply:
Whoops, didn't consider that `\w` doesn't include `-` but does include `_` for some weird reason.